README.md

```diff
@@ -43,6 +43,53 @@ Installation
     ```
 
 
+Developing
+----------
+
+1. Clone the git repository:
+   ```sh
+       $ git clone https://github.com/unt-libraries/django-citeit.git
+   ```
+
+2. Navigate into the cloned repository:
+   ```sh
+       $ cd django-citeit
+   ```
+
+3. Install the requirements (preferably in a virtual environment):
+   ```sh
+       $ pip install -r requirements-dev.txt
+   ```
+
+4. Navigate to the root of the project directory and run the migrations:
```

Comment:
We don't need the "Navigate to the root of the project directory and " part, since we should already be there based on the previous two commands.